crypto/keys/private.go

```diff
@@ -38,8 +38,11 @@ type SignerFactory interface {
 }
 
 // NewFromPrivatePEMFile reads a PEM-encoded private key from a file.
-// The key may be protected by a password.
+// The key must be protected by a password.
 func NewFromPrivatePEMFile(keyFile, keyPassword string) (crypto.Signer, error) {
+	if keyPassword == "" {
+		return nil, errors.New("empty PEM key password")
```

Comment:
Could you include the `keyFile` value in the error message please, to make it obvious which file needs a password?